sormas-ui/src/main/java/de/symeda/sormas/ui/events/EventGroupController.java

```diff
@@ -203,19 +202,21 @@ public CommitDiscardWrapperComponent<?> getEventGroupEditComponent(String uuid)
 			});
 		}
 
-		if (user.hasUserRight(UserRight.EVENTGROUP_DELETE) && hasRegion) {
-			editView.addDeleteListener(() -> {
-				deleteEventGroup(eventGroup);
-				UI.getCurrent().getNavigator().navigateTo(EventsView.VIEW_NAME);
-			}, I18nProperties.getString(Strings.entityEventGroup));
-		}
+		// TODO Temporarily deleted in #8915, enable again in #8851
+//		if (user.hasUserRight(UserRight.EVENTGROUP_DELETE) && hasRegion) {
+//			editView.addDeleteListener(() -> {
+//				deleteEventGroup(eventGroup);
+//				UI.getCurrent().getNavigator().navigateTo(EventsView.VIEW_NAME);
+//			}, I18nProperties.getString(Strings.entityEventGroup));
+//		}
 
 		// Initialize 'Archive' button
 		if (user.hasUserRight(UserRight.EVENTGROUP_ARCHIVE) && hasRegion) {
 			boolean archived = FacadeProvider.getEventGroupFacade().isArchived(uuid);
-			Button archiveEventButton = ButtonHelper.createButton(archived ? Captions.actionDearchiveInfrastructure : Captions.actionArchiveInfrastructure, e -> {
-				archiveOrDearchiveEventGroup(uuid, !archived);
-			}, ValoTheme.BUTTON_LINK);
+			Button archiveEventButton =
```

Comment:
Better not format something else: When you keep it clean, a revert action can work.

Reply:
This has been done by automatic formatting; manually re-formatting it in the wrong way doesn't seem like a good idea to me.

Reply:
Just disable the automatic formatting for this change to not change something unintended. That's the problem when not everyone formats the code the same way :-(